service/src/main/kotlin/fi/espoo/evaka/pairing/MobileDeviceQueries.kt

```diff
@@ -0,0 +1,68 @@
+package fi.espoo.evaka.pairing
+
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.NotFound
+import org.jdbi.v3.core.kotlin.mapTo
+import java.util.UUID
+
+fun insertDeviceData(tx: Database.Transaction, unitId: UUID): UUID {
+    val defaultName = "Nimeämätön laite"
+
+    // language=sql
+    val employeeInsert =
+        """
+            INSERT INTO employee (first_name, last_name, email, aad_object_id)
```

Comment:
Personal preference: I'd combine these queries into one. Something like this:

```
INSERT INTO mobile_device (id, unit_id, name)
SELECT id, :unit_id, :name
FROM (
  INSERT INTO employee (...)
  VALUES (...)
  RETURNING id
) new_employee
```

Reply:
couldn't get that syntax to work so went with
```
WITH new_employee AS (
    INSERT INTO employee (first_name, last_name, email, aad_object_id)
    VALUES (:name, (SELECT name FROM daycare WHERE id = :unitId), null, null)
    RETURNING id
)
INSERT INTO mobile_device (id, unit_id, name) 
SELECT new_employee.id, :unitId, :name
FROM new_employee
RETURNING id
```